README.md

```diff
@@ -1,6 +1,23 @@
 # Welcome to Crowbar Angular Prototype
 [![Build Status](https://travis-ci.org/crowbar/crowbar-ui.svg?branch=master)](https://travis-ci.org/crowbar/crowbar-ui)
 
+### Setting up a development environment with Docker
+
+1. make sure `docker` is installed and the `docker` daemon is running.
+2. build the image from the [Dockerfile](Dockerfile):
+```
+docker build -t crowbar-ui .
+```
+3. run the container:
+```
+docker run -d --name "crowbar-ui_container" -v $(pwd)/assets:/crowbar-ui/assets -p 3000:3000 crowbar-ui
+```
+4. run the `gulp` watchdog:
+```
+docker exec -ti crowbar-ui_container gulp
+```
+5. work on the assets folder in your local git clone and the changes will be immediately visible in the browser.
```

Comment:
the problem here is that the testing creates a subdirectory under `/coverage/PhantomJS\ 2.1.1\ \(Linux\ 0.0.0\)`. it can be served by `express` if the folder is registered as a static folder. but how can i know this value? it seems to be dynamic (with the version of phantomjs)

it should be something like:
```js
if (app.get('env') === 'development') {
    app.use(express.static(__dirname + '/coverage/' + get_name_of_that_directory()));
}
```

then it could easily be accessed via `localhost:3000/coverage`